config/recipes/beats/README.md

```diff
@@ -0,0 +1,39 @@
+# Beats Configuration Library
+
+This directory contains yaml manifests with example configurations for Beats. These manifests are self-contained and work out-of-the-box on any non-secured Kubernetes cluster. All of them contain three-node Elasticsearch cluster and single Kibana instance. All Beat configurations set up Kibana dashboards if they are available for a given Beat and all required RBAC resources. 
```

Comment:
Should we mention how the Elasticsearch spec is not suitable to a production environment?

Reply:
This is already there in https://github.com/elastic/cloud-on-k8s/blob/master/config/recipes/README.asciidoc, should we just repeat it here?

Reply:
I'm not sure people will notice the root README, it's worth repeating here imo.